lib/pacemaker/pcmk_cluster_queries.c

```diff
@@ -73,7 +73,7 @@ event_done(gpointer user_data, pcmk_ipc_api_t *api)
 {
     data_t *data = user_data;
     pcmk_disconnect_ipc(api);
-    quit_main_loop(data, data->exit_code);
+    quit_main_loop(data, data->rc);
```

Comment:
I'm thinking with the new struct, there's no point in having quit_main_loop() take the error code. The one call can set it directly, and this call doesn't need it.